js/state/siteUtil.js

```diff
@@ -747,3 +749,7 @@ module.exports.setObjectId = (item) => {
   const crypto = require('crypto')
   return item.set('objectId', new Immutable.List(crypto.randomBytes(16)))
 }
+
+module.exports.enableNoScript = (settings, state) => {
```

Comment:
siteUtil.js is actually for `appState.sites` and site settings are in `appState.siteSettings`. The names are similar, but they are two very distinct things. Maybe a new `siteSettingsState` in `app/common/state`?

Also this method sounds like it will enable noScript, but it's actually checking the state of noScript so maybe `noScriptEnabled` or `isNoScriptEnabled`? Can we swap also swap the state and settings parameters to be consistent with other state helpers and maybe move this to `siteSettingsState`?